### PR TITLE
Verlet

### PR DESCRIPTION
**Pull request by Ved-Mahajan**

### CODE REVIEW

File: src/TheAlgorithms.jl [hunk 1]
```diff
@@ -149,6 +157,8 @@ include("project-rosalind/count_nucleotide.jl")
 include("project-rosalind/dna2rna.jl")
 include("project-rosalind/reverse_complement.jl")
 
+# Includes: scheduling
+
```
Comment (Panquesito7) [inline src/TheAlgorithms.jl:161]:
```suggestion
```

File: src/math/monte_carlo_integration.jl [hunk 2]
```diff
@@ -0,0 +1,20 @@
+"""
+Monte carlo integration is a very easy and scalable way to do multidimentional integrals.
+However, only single variable integrals is considered.
+
+
+This function takes three outputs:
```
Comment (Panquesito7) [inline src/math/monte_carlo_integration.jl:6]: Please add proper documentation like in the [`knapsack`](https://github.com/TheAlgorithms/Julia/blob/main/src/knapsack/knapsack.jl) problem.

**Review by Panquesito7 — COMMENTED** [review]
Please make one PR per file. I see you have added the Monte Carlo integration, Sieve of Eratosthenes, and Verlet, too. Please make separate PRs per file.

**Review by arubhardwaj — CHANGES_REQUESTED** [review]
Have a look at why CI failed. I tried finding but seems like some other issue

**Comment by Ved-Mahajan** [comment]
> Have a look at why CI failed. I tried finding but seems like some other issue

you mean those yellow dots?

I am also clueless as to why have they not started yet

File: src/math/monte_carlo_integration.jl (hunk 2 above)
Comment (Ved-Mahajan) [reply src/math/monte_carlo_integration.jl:6]: Done in the monte carlo PR

**Comment by Ved-Mahajan** [comment]
> Please make one PR per file. I see you have added the Monte Carlo integration, Sieve of Eratosthenes, and Verlet, too. Please make separate PRs per file.

I noticed that after pushing..the code, however, is the same as the other PRs so merginng should not be harmful

It was my mistake as the tests failed and I got confused why did it failed.

**Comment by Panquesito7** [comment]
> you mean those yellow dots?

Sorry, I made something wrong with the branch protection. Fixed though. 😅

**Review by Azzaare — APPROVED** [review]
This seems good to go.

**Comment by Ved-Mahajan** [comment]
I am closing this PR for the huge mount of mess in there

**Pull request closed without merging**